### DISCUSSION TITLE
Scrolling sensitivity feels different

### DISCUSSION THREAD

**Discussion by nikandfor** [community/community#17302] [Codespaces] [answered]
I've noticed shortly the scrolling speed is different on the codespace page. Both the editor panel and the tools panels scrolls twice as fast as all the other pages. This is confusing.

I use the Macbook Air M1 (MacOS 12.3.1) and the Google Chrome (101.0.4951.64 (Official Build) (arm64)).

**Comment by legomushroom** [comment, marked as answer]
hey @nikandfor 👋 Thanks for reporting! That is a vscode issue with trackpad scrolling not with Codespaces per se, chiming in on this issue so we keep everything in one place?

👉 https://github.com/microsoft/vscode/issues/148217
